javascripts/discourse/components/custom-header-dropdown.gjs

```diff
@@ -25,6 +25,7 @@ export default class CustomHeaderDropdown extends Component {
     <li
       class="custom-header-dropdown-link"
       title={{@item.title}}
+      role="button"
```

Comment:
This fixes the "_Interaction added to non-interactive element_" error.
https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-invalid-interactive.md

Ideally, I think we want to use `<a>` link here for better accessibility. 
That's something I will see to change in another PR.